dom/base/nsContentUtils.cpp

```diff
@@ -7760,6 +7760,13 @@ nsContentUtils::SendMouseEvent(nsCOMPtr<nsIPresShell> aPresShell,
     msg = eMouseUp;
   } else if (aType.EqualsLiteral("mousemove")) {
     msg = eMouseMove;
+
+    // If need to draw mouse cursor, set point then force composition
+    if(widget->GetDrawMouse()) {
+      ScreenIntPoint point(floor(aX+0.5), floor(aY+0.5));
+      widget->SetScreenIntPoint(point);
+      widget->ForceComposition();
+    }
```

Comment:
It would be better to scope this block with a #ifdef MOZ_WIDGET_GONK, since ForceComposition() is a API only for Gonk::nsWindow.